src/init.cpp

```diff
@@ -295,7 +295,7 @@ std::string HelpMessage(HelpMessageMode mode)
     strUsage += HelpMessageOpt("-conf=<file>", strprintf(_("Specify configuration file (default: %s)"), "bitcoin.conf"));
     if (mode == HMM_BITCOIND)
     {
-#if !defined(WIN32)
+#ifndef WIN32
```

Comment:
What is the point of this change? `#if !defined(XX)` and `#ifndef XX` are equivalent, with the difference that the first can be used as part of expressions, but I don't see a reason to change it around.